internal/bot.py

```diff
@@ -55,6 +54,7 @@ async def login(self, *args, **kwargs) -> None:
         await super().login(*args, **kwargs)
 
     async def get_prefix(self, message: Message) -> str:
+        '''
```

Comment:
This shouldn't be commented out

Reply:
yeah I didnt mean to commit it while that was commented, mb.